packages/scenes/src/querying/SceneQueryRunner.ts

```diff
@@ -108,7 +109,7 @@ export class SceneQueryRunner extends SceneObjectBase<QueryRunnerState> implemen
   private _containerWidth?: number;
   private _variableValueRecorder = new VariableValueRecorder();
   private _results = new ReplaySubject<SceneDataProviderResult>(1);
-  private _scopedVars = { __sceneObject: { value: this, text: '__sceneObject' } };
+  private _scopedVars = { __sceneObject: { value: new SafeSerializableSceneObject(this), text: '__sceneObject' } };
```

Comment:
```suggestion
  private _scopedVars = { __sceneObject: new SafeSerializableSceneObject(this) };
```

@dprokop I have an idea that could make this a tiny bit cleaner and reduce some code duplication. We just have to make  SafeSerializableSceneObject have two public properties (value & text), value could just be an empty string, and text would be "__sceneObject". 

Then it becomes a bit cleaner to use and we do not have to duplicate ` { __sceneObject: { value: new SafeSerializableSceneObject(this), text: '__sceneObject' } };` everywhere.

Reply:
`value` cannot be an empty string- in core we actually need an access to this object itself for interpolation :) But i've simplified the code a bit, value now returns identity, and there's `text` available as well so indeed less duplication overall.